kbase-extension/static/kbase/js/common/cellComponents/filePathWidget.js

```diff
@@ -183,7 +180,11 @@ define([
             fieldWidget.bus.on('validation', (message) => {
                 // only propagate if invalid. value changes come through
                 // the 'changed' message
-                if (!message.isValid) {
+                let duplicateValues = [];
+                if (model.getItem('outputParamIds').includes(parameterSpec.id)) {
```

Comment:
This is duplicated-ish from the `changed` response.
It turns out that output parameter widgets send both a "changed" and a "validation" response when changes happen. The changes are intended to be stored whether they're an error or not. Validation is to say whether that's valid or not.

The problem comes at widget startup - both of those messages are fired more or less at the same time. So catching one to do the duplication check could get overridden by the other. It's a little annoying, but clean enough, to do the check twice.

Another solution would probably be to either refactor the input widget itself (which would mean an overhaul of all the input widget behaviors), or to cache what values are actually duplicates somewhere and try to keep that updated as we go... which is also a ton more code to fix what I think is a message-spam issue.